### PR TITLE
Move Recon service to mixer

### PR DESCRIPTION
**Pull request by shifucun**
- Add two new flags to indicate whether to serve Mixer and/or Recon service in main.go. Mixer service is enabled and Recon service is disabled by default.
- Copy over existing Recon code from https://github.com/datacommonsorg/reconciliation/tree/d8e67c65dbf94cda1c7a95e1b685ce6576a2345e
- In a follow up PR, will update the GKE deployment config for both servers.

### CODE REVIEW

**Review by beets — COMMENTED** [review]
qq: instead of copying over all the recon code, should we try a submodule? it's worked quite well for website, and it still leaves the possibility of a standalone recon service plus it isolates changes there somewhat. wdyt?

**Comment by shifucun** [comment]
> qq: instead of copying over all the recon code, should we try a submodule? it's worked quite well for website, and it still leaves the possibility of a standalone recon service plus it isolates changes there somewhat. wdyt?

The recon and mixer are tightly coupled here, from proto to server impl to deploy config and they share many deps. If including as a submodule, all the current recon codes would be included, as compared now only a handful (~10 files) are needed.

In the website situation, mixer is an API server which is quite standalone, but in this case we sort of run one server with multiple services: mixer and/or recon service

**Review by beets — APPROVED** [review]
gotcha - sounds reasonable. it's really just the service portion that we need to include here.

File: internal/server/handler.go [hunk 1]
```diff
@@ -266,3 +267,40 @@ func (s *Server) GetVersion(
 		GitHash:  os.Getenv("MIXER_HASH"),
 	}, nil
 }
+
+// GetVersion implements API for Mixer.GetVersion.
```
[the remark this answers is not in the document]
Comment (shifucun) [reply internal/server/handler.go]: Good catch!

**Review by Spaceenter — APPROVED** [review]

**Pull request merged**